pkgs/applications/misc/ebook2cw/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, lame, libvorbis }:
+
+stdenv.mkDerivation rec {
+
+  name = "ebook2cw-${version}";
+  version = "0.8.2";
+
+  src = fetchurl {
+    url = "http://fkurz.net/ham/ebook2cw/${name}.tar.gz";
+    sha256 = "0n2p1i8cnnl2y3cdw8knx1vvs39sid45fr59p6f4vrprmb37l9yq";
+  };
+
+# TODO: Use SVN once fetchsvn stops stripping the trailing slash,
+# otherwise we get a build failure due to a redirect cycle.
+#
+#  src = fetchsvn {
+#    url = "http://svn.fkurz.net/ebook2cw/tags/${name}/";
+#    sha256 = "0n2p1i8cnnl2y3cdw8knx1vvs39sid45fr59p6f4vrprmb37l9yq";
+#  };
+
+  buildInputs = [ lame libvorbis ];
+
+  patchPhase = "patch -p0 < ${./configfile.patch}";
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  meta = with stdenv.lib; {
+    description = "Convert ebooks to Morse MP3s/OGGs";
+    homepage = http://fkurz.net/ham/ebook2cw.html;
+    license = licenses.gpl2;
+    platforms = platforms.linux;
```

Comment:
This should build on other platforms as well, but I have no way to test it.

Reply:
Since upstream supports other platforms, I'd bump to `platforms.all`.

Reply:
👍